capstone/src/test/java/com/google/sps/LessonTest.java

```diff
@@ -31,8 +31,7 @@
 
     //Helper to set local datastore service
     private final LocalServiceTestHelper helper =
-      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig()
-          .setDefaultHighRepJobPolicyUnappliedJobPercentage(100));
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
```

Comment:
Since this is a completely unrelated change it should not be part of this PR. Its fine to create a 1 line PR if this is not related to anything else at all, but having it piggyback as part of another CL can cause problems if for example this CL caused a bug and needed to be rolled back

Reply:
Sounds good. I just thought having a 1 line PR seems really minuscule but this makes a lot more sense.